packages/atlas/src/views/viewer/NftsView/NftsView.tsx

```diff
@@ -134,7 +134,11 @@ export const NftsView: FC = () => {
           </div>
           {smMatch && sortingNode}
         </HeaderContainer>
-        <FiltersBar {...filtersBarLogic} activeFilters={['nftStatus', 'date-minted', 'other']} />
+        <FiltersBar
```

Comment:
should we add the `onAnyFilterSet` for the other `FiltersBar` component usage in a separate PR?

Reply:
I think so, most of the components that use FilterBar are using it in a way that makes it impossible to handle it with just onAnyFilterChange tho. 

Flow is reversed, so Filterbar is implemented in the parent component while pagination lives strictly inside children (most often it's tab content), so the only way I see to enforce pagination reset on children when we have an event in the parent is either `useEffect` when `totalCount / filters` change or using `useImperativeHandle`.